cfme/tests/infrastructure/test_timelines.py

```diff
@@ -1,242 +1,128 @@
+            self.db_event = ''
+            self.emit_cmd = self._setup_vm()
+        elif self.action in {'start', 'resume'}:
+            self.tl_event = 'VmPoweredOnEvent'
+            self.tl_category = 'Power Activity'
+            self.db_event = ''
+            self.emit_cmd = self._power_on()
+        elif self.action == 'stop':
+            self.tl_event = 'VmPoweredOffEvent'
+            self.tl_category = 'Power Activity'
+            self.db_event = ''
+            self.emit_cmd = self._power_off()
+        elif self.action == 'suspend':
+            self.tl_event = 'VmSuspendedEvent'
+            self.tl_category = 'Power Activity'
+            self.db_event = ''
+            self.emit_cmd = self._suspend()
+        elif self.action == 'rename':
+            self.tl_event = 'VmReconfiguredEvent'
+            self.tl_category = 'Configuration/Reconfiguration'
+            self.db_event = ''
+            self.emit_cmd = self._rename_vm()
+        elif self.action == 'storage':
+            self.tl_event = 'VmReconfiguredEvent'
+            self.tl_category = 'Storage'
+            self.db_event = ''
+            self.emit_cmd = self._rename_vm()
         else:
-            responses = appliance.rest_api.collections.vms.action.add_lifecycle_event(
-                rest_vm, **event)
-        assert appliance.rest_api.response.status_code == 200
-        for response in responses:
-            assert response["success"] is True, "Could not add event"
-
-        # DB check
-        lifecycle_events = appliance.db.client["lifecycle_events"]
-        events_list = list(appliance.db.client.session.query(lifecycle_events).filter(
-            lifecycle_events.message == event["message"],
-            lifecycle_events.status == event["status"],
-            lifecycle_events.event == event["event"],
-        ))
-        assert events_list, "Could not find the lifecycle event in the database"
-
-
-@pytest.mark.manual
-def test_policy_events():
-    pass
-
-
-@pytest.mark.manual
-def test_timelines_ui():
-    pass
+            assert False, logger.warn('VMEvent did not get the proper parameter.')
+
+    def _setup_vm(self):
+        logger.info("Will set up the VM on the provider")
+        self.vm.create_on_provider()
+        return self.vm.provider.mgmt.does_vm_exist(self.vm.name)
+
+    def _power_on(self):
+        logger.info('{} will start.'.format({self.vm.name}))
+        return self.vm.provider.mgmt.start_vm(self.vm.name)
+
+    def _power_off(self):
+        logger.info('{} will stop.'.format({self.vm.name}))
+        return self.vm.provider.mgmt.stop_vm(self.vm.name)
+
+    def _suspend(self):
+        logger.warn("Will Suspend {}!".format(self.vm.name))
+        return self.vm.provider.mgmt.suspend_vm(self.vm.name)
+
+    def _rename_vm(self):
+        logger.warn("Will delete {}!".format(self.vm.name))
+        new_name = self.vm.provider.mgmt.rename_vm(self.vm.name, self.vm.name + '-renamed')
+        if self.vm.name != new_name:
```

Comment:
OPTIONAL: please make sure you need such change at all

Reply:
It is to have some extra event from Power events. It is in this case some Configuration event